.github/workflows/comments.yml

```diff
@@ -109,6 +117,7 @@ jobs:
             Write-Host "Offensive: $offensive"
 
             if ($anger -ge 7 -and $offensive -ge 7) {
```

Comment:
This isn't this PR, but maybe we should also put the number 7 as a configurable valeu.